Gemeinsamer Bibliotheksverbund ISBN.js

```diff
@@ -0,0 +1,238 @@
+				}];
+			}
+			//DDC is not the callNumber in Germany
+			item.callNumber = "";
+			item.complete();
+		});
+		translator.translate();
+
+	});
+}/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"978-3-642-00230-4\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Eigentumsverfassung und Finanzkrise",
+				"creators": [
+					{
+						"firstName": "Otto",
+						"lastName": "Depenheuer",
+						"creatorType": "editor"
+					},
+					{
+						"lastName": "Symposion. Deutsche Stiftung Eigentum",
+						"fieldMode": true
+					}
+				],
+				"date": "2009",
+				"ISBN": "9783642002304",
+				"language": "ger",
+				"place": "Berlin ;Heidelberg",
+				"publisher": "Springer",
+				"series": "Bibliothek des Eigentums, Im Auftrag der Deutschen Stiftung Eigentum",
+				"seriesNumber": "7",
+				"attachments": [],
+				"tags": [
+					"Aufsatzsammlung",
+					"Aufsatzsammlung",
+					"Constitutional law",
+					"Constitutional law",
+					"Constitutional law",
+					"Eigentum",
+					"Finanzkrise",
+					"Haftung",
+					"Law",
+					"Law",
+					"Law",
+					"Online-Publikation",
+					"Online-Publikation",
+					"Ordnungspolitik"
+				],
+				"notes": [
+					{
+						"note": "\"Unter dem Thema 'Eigentumsverfassung und Finanzkrise' veranstaltete die Deutsche Stiftung Eigentum am 22. April 2009 in Berlin ein Symposion"
+					},
+					{
+						"note": "Die weltweite Finanzkrise ist Anlass, an Funktion und Wirkweise des privaten Eigentums in einer freiheitlichen Gesellschafts- und Wirtschaftsordnung zu erinnern. Privates Eigentum muss es geben, damit Verantwortung zugerechnet und Haftung realisiert, Gewinn und Verlust einem konkreten Verantwortungsträger persönlich zugerechnet werden können. Die Verletzung dieser konstitutiven Regeln einer auf privatem Eigentum basierenden Wirtschaftsordnung ist wesentlich ursächlich für das eingetretene Desaster auf den Finanzmärkten. Wie alle kulturellen Errungenschaften muss auch die Idee des privaten Eigentums, insbesondere die ihr immanente Bereitschaft zur Übernahme persönlicher Verantwortung des Eigentümers, jeder Generation erneut wieder in Erinnerung gerufen, überzeugend um sie geworben und vor allem vorbildhaft von den Akteuren in Politik und Wirtschaft vorgelebt werden. Nur so kann strukturelles Vertrauen in das Finanzsystem wieder gewonnen werden. Denn in ihrer vertrauensbildenden Kraft liegt die ordnungspolitische Funktion der Gewährleistung privaten Eigentums"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"3-86688-240-8\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Bilinguale Lexik nicht materieller lexikalischer Transfer als Folge der aktuellen russisch-deutschen Zweisprachigkeit",
+				"creators": [
+					{
+						"firstName": "Katrin",
+						"lastName": "Karl",
+						"creatorType": "author"
+					}
+				],
+				"date": "2012",
+				"ISBN": "9783866882416",
```

Comment:
So I'm not really clear on what the meaning of multiple ISBNs is. Is it different formats of publication? Different publishers? Wouldn't the metadata be different for different ISBNs then?

In general I want to say yes as well. The only thing I would do is (1) verify that the queried ISBN is in the list and reject the results that do not contain the query (not sure if that can actually happen with GBV) and (2) place the queried ISBN as the first ISBN in the list (you don't have to worry about removing the same ISBN from somewhere else in the list)

What we can do from Zotero's side is:
(1) On import from translators (or even editing the field), validate the ISBNs in the list, convert all valid ISBNs to ISBN-13, hyphenate them (unless this is style-specific), and get rid of any duplicates
(2) Pass the first ISBN from the list to citeproc
(3) Let users search by ISBN-10 and ISBN-13

We need to decide what separator to use for ISBNs. I'm thinking either space, comma, or semicolon. I think all of those are being used currently. Semicolon is probably the safest.